pkg/operator/resource/resourceapply/storage.go

```diff
@@ -115,6 +120,10 @@ func ApplyCSIDriver(ctx context.Context, client storageclientv1.CSIDriversGetter
 	if err != nil {
 		return nil, false, err
 	}
+	err = validateRequiredCSIDriverLabels(required)
```

Comment:
Nit: I like when those are written like so:
```golang
if err = validateRequiredCSIDriverLabels(required); err != nil {
```
it makes them more readable and can potentially save from shadowing errors too.

Reply:
done